### ISSUE TITLE
Do Not Disqualify on Approval

### ISSUE THREAD

**Issue by 0x4007** [ubiquity-os-marketplace/daemon-disqualifier#57]
[daemon-merging](https://github.com/ubiquity-os-marketplace/daemon-merging) doesn't have a chance to run properly if disqualifications happen on approved pulls. This should be fixed by halting the disqualifier logic once they received an approval. 

Even if an argument is made around "increasing coupling" it still doesn't make sense to disqualify somebody after their deliverable has been approved, and there are delays from the other reviewers. 

My gut feeling is that disqualification should continue to run on review states for "commented" and "request changes" 

Perhaps in the future, a minor optimization could be to enforce that the conversations from the "commented" review must be resolved in order to halt disqualification, but this is easily gamed as I've seen assignees "resolve" comments without actually handling them. We would need to have more advanced logic to compare the diff with the review comments to see if they were actually addressed. 

---

https://t.me/UbiquityDevPool/2990

Passed the deadline and no activity is detected, removing assignees: @kingsley-einstein.

_Originally posted by @ubiquity-os[bot] in https://github.com/ubiquity/uusd.ubq.fi/issues/1#issuecomment-2515349740_

**Comment by 0x4007** [comment]
Yeah the most high tech implementation would understand that there was activity from the assignee even on another pull request. 

That means it would need to be aware of activity on:

1. The issue
2. The current pull
3. Any other associated pull.

**Comment by whilefoo** [comment]
> @whilefoo you have used all available deadline extensions and have been disqualified from this task.
> 
> 
>  

 _Originally posted by @ubiquity-os-beta in [#23](https://github.com/ubiquity-os-marketplace/daemon-pull-review/issues/23#issuecomment-2739822030)_

I was un-assigned even though the PR has 2 approvals

**Comment by 0x4007** [comment]
@gentlementlegen  why closed as unplanned?

**Comment by gentlementlegen** [comment]
> I think this is not exactly related to this, it seems to be that the disqualifier considers the wrong issue when updating the state of the reminder, which was mentioned before (although it's just a guess).

It seems that it was linked to this behavior, which was fixed, I'll reopen if that happens again!

**Comment by ishowvel** [comment]
it has @gentlementlegen https://github.com/ubiquity/work.ubq.fi/pull/212#issuecomment-2878459631

**Comment by gentlementlegen** [comment]
It is supposed to send reminder, as long as it doesn't disqualify it is intended.
